libs/ojsonnet/Eval_jsonnet.ml

```diff
@@ -670,6 +668,22 @@ and manifest_value (v : V.value_) : JSON.t =
                    let new_self = V.Object (_l, (_assertsTODO, fields), _r) in
                    let locals =
                      fld_value.env.locals
+                     (* We need to do these assignment on field access rather than on
```

Comment:
I think this comment would be better in eval_expr in the field access code rather than here in manifest_value.